lib/vm/opFns.js

```diff
@@ -602,7 +602,7 @@ module.exports = {
     var options = {
       value: value,
       data: data,
-      salt: salt.toBuffer('be', 32)
+      salt: utils.toBuffer(salt)
```

Comment:
Not specifying the buffer size seems to have caused this [assertion](https://github.com/ethereumjs/ethereumjs-util/blob/8ffe697fafb33cefc7b7ec01c11e3a7da787fe0e/src/account.ts#L93) to fail. We should consider adding a size parameter to util's `toBuffer`, in the meanwhile could you directly use `toArrayLike` also providing the endianness and size?

Reply:
oh you are right, `toArrayLike` is the one to use yes